artio-core/src/main/java/uk/co/real_logic/artio/engine/PositionSender.java

```diff
@@ -96,4 +96,11 @@ public void close()
     public void readLastPosition(final IndexedPositionConsumer consumer)
     {
     }
+
+    @Override
```

Comment:
We don't use @Override in real logic projects.

Reply:
removed